### ISSUE TITLE
Home Assistant restart loop after upgrade to 3.4.3

### ISSUE THREAD

**Issue by massaquah** [pnbruckner/ha-composite-tracker#82]
After upgrading composite from 3.4.2 to 3.4.3 and restarting Home Assistant I get a reproducible restart loop.

```
2024-08-17 09:06:05.379 ERROR (MainThread) [root] Uncaught exception
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/usr/src/homeassistant/homeassistant/__main__.py", line 223, in <module>
    sys.exit(main())
             ^^^^^^
  File "/usr/src/homeassistant/homeassistant/__main__.py", line 209, in main
    exit_code = runner.run(runtime_conf)
                ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/runner.py", line 190, in run
    return loop.run_until_complete(setup_and_run_hass(runtime_config))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/runner.py", line 158, in setup_and_run_hass
    hass = await bootstrap.async_setup_hass(runtime_config)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/bootstrap.py", line 314, in async_setup_hass
    config_dict = await conf_util.async_hass_config_yaml(hass)
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/config.py", line 486, in async_hass_config_yaml
    await merge_packages_config(hass, config, core_config.get(CONF_PACKAGES, {}))
  File "/usr/src/homeassistant/homeassistant/config.py", line 1102, in merge_packages_config
    merge_list = config_platform.PACKAGE_MERGE_HINT == "list"
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'custom_components.composite.config' has no attribute 'PACKAGE_MERGE_HINT'
```

**Comment by pnbruckner** [comment]
Sorry, my bad. I'll release 3.4.4 shortly with a fix.

**Comment by pnbruckner** [comment]
Released 3.4.4. Let me know if it fixes the problem for you.

**Comment by massaquah** [comment]
Seems to work.
At least no restart loop anymore.
Thank you.
